src/modules/app/containers/__tests__/I18nRedirect.spec.js

```diff
@@ -12,7 +12,7 @@ describe('I18nRedirect', () => {
       <I18nRedirect
         currentPath={'/augsburg/'}
         redirect={redirect} />,
-      {context: {i18n: {language: 'de'}}, childContextTypes: {i18n: PropTypes.object.isRequired}} // fixme why can't we use I18nProvider here?
+      {context: {i18n: {language: 'de'}}, childContextTypes: {i18n: PropTypes.object.isRequired}}
```

Comment:
this is okay, see the official doc: https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md#arguments

Reply:
Yes it works, but it still makes no sense that wrapping it in a I18nProvider does not work here.

Reply:
yep, I tried, but couldn't get it to work...should I create a task for this?

Reply:
Yep, this "fixme" is not fixed yet :P Maybe there are reasons why this does not work. Or it's a but of enzyme

Reply:
done